pkg/security/resolvers/sbom/resolver.go

```diff
@@ -195,7 +194,7 @@ func (r *Resolver) generateSBOM(root string, sbom *SBOM) error {
 	seclog.Infof("Generating SBOM for %s", root)
 	r.sbomGenerations.Inc()
 
-	report, err := r.trivyScanner.ScanFilesystem(context.Background(), root)
+	report, err := r.trivyScanner.ScanFilesystem(context.Background(), root, sbompkg.ScanOptions{Analyzers: []string{trivy.OSAnalyzers}})
```

Comment:
Why not using the shared scanner?

Reply:
Because we are not in the same process, so we won't share anything